app/src/androidTest/java/com/sdp/movemeet/HomeScreen/HomeScreenActivityUnloggedTest.java

```diff
@@ -47,49 +54,14 @@ public void tearDown() {
 
     @Test
     public void signInHasCorrectIntentWhenUnlogged()  {
-        fAuth = FirebaseAuth.getInstance();
-        if (fAuth.getCurrentUser() != null) fAuth.signOut();
-        sleep(2000);
         onView(withId(R.id.signInButton)).perform(click());
-        sleep(2000);
         intended(hasComponent(LoginActivity.class.getName()));
     }
 
     @Test
-    public void signInHasCorrectIntentWhenLogged()  {
```

Comment:
Why do you delete this two test ?

Reply:
I only moved them to a new class dedicated to when the user is logged "HomeScreenActivityLoggedTest". I find it more clear this way, and it's also easier to test with the firebase authentication :)